core/templates/services/utils.service.ts

```diff
@@ -19,29 +19,27 @@
 
 import { Injectable } from '@angular/core';
 import { downgradeInjectable } from '@angular/upgrade/static';
-
 @Injectable({
   providedIn: 'root'
 })
 export class UtilsService {
   /**
-   * Determines if a variable is defined and not null.
-   * @param {Object, Array<Object>, string, string[], undefined, null}value
-   * @return {boolean} - true if object is defined, false otherwise.
-   */
+  * Determines if a variable is defined and not null.
+  * @param {Object, Array<Object>, string, string[], undefined, null}value
+  * @return {boolean} - true if object is defined, false otherwise.
+  */
   isDefined(
       value: Object | Object[] | string | string[] | undefined | null
   ): boolean {
     return typeof value !== 'undefined' && value !== null;
   }
-
   // The function here is to check whether the argument is empty or not. So, we
   // cannot have any specific type defined for the argument and the argument
   // is given a generic type of Object.
   /**
-   * @param {Object} obj - the object to be checked.
-   * @return {boolean} - true if object is empty, false otherwise.
-   */
+  * @param {Object} obj - the object to be checked.
+  * @return {boolean} - true if object is empty, false otherwise.
```

Comment:
Undo these changes*

Reply:
Done!

Reply:
In the commit "_removing unwanted changes from core/templates/services/utils.service.ts_" I restored the spaces so that they no longer differ from the develop branch.

Reply:
No, you didn't this file still differs from develop.

Reply:
Yeah. Sorry, I didn't notice the indentation problem. But now it's Done!